### ISSUE TITLE
Error when running dockerPush

### ISSUE THREAD

**Issue by jkinkead** [allenai/sbt-plugins#278]
Running `dockerPush` resulted in the following error:

```
[Step 2/2] Step 1 : FROM allenai-docker-private-docker.bintray.io/ari/evaluation-ui-dependencies
*snip* docker build logs...
[19:05:48]	[Step 2/2] Step 11 : COPY lib lib
[19:05:48]	[Step 2/2]  ---> f87727b7f4ab
[19:05:48]	[Step 2/2] Removing intermediate container e5079a5e4dd0
[19:05:48]	[Step 2/2] Successfully built f87727b7f4ab
[19:05:48]	[Step 2/2] [info] Removing stale image...
[19:05:48]	[Step 2/2] Untagged: allenai-docker-private-docker.bintray.io/ari/evaluation-ui:e175f15e7bd2df7c7b084b5d840a2874d4436f89
[19:05:48]	[Step 2/2] The push refers to a repository [allenai-docker-private-docker.bintray.io/ari/evaluation-ui]
[19:05:48]	[Step 2/2] java.lang.RuntimeException: There was a problem pushing tag "e175f15e7bd2df7c7b084b5d840a2874d4436f89", stopping.
[19:05:48]	[Step 2/2] 	at scala.sys.package$.error(package.scala:27)
[19:05:48]	[Step 2/2] 	at org.allenai.plugins.DockerBuildPlugin$$anonfun$dockerPushDef$1$$anonfun$apply$13$$anonfun$apply$14.apply(DockerBuildPlugin.scala:819)
[19:05:48]	[Step 2/2] 	at org.allenai.plugins.DockerBuildPlugin$$anonfun$dockerPushDef$1$$anonfun$apply$13$$anonfun$apply$14.apply(DockerBuildPlugin.scala:786)
[19:05:48]	[Step 2/2] 	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
[19:05:48]	[Step 2/2] 	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
[19:05:48]	[Step 2/2] 	at sbt.std.Transform$$anon$4.work(System.scala:63)
[19:05:48]	[Step 2/2] 	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
[19:05:48]	[Step 2/2] 	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
[19:05:48]	[Step 2/2] 	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
[19:05:48]	[Step 2/2] 	at sbt.Execute.work(Execute.scala:237)
[19:05:48]	[Step 2/2] 	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
[19:05:48]	[Step 2/2] 	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
[19:05:48]	[Step 2/2] 	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
[19:05:48]	[Step 2/2] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
[19:05:48]	[Step 2/2] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[19:05:48]	[Step 2/2] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[19:05:48]	[Step 2/2] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[19:05:48]	[Step 2/2] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[19:05:48]	[Step 2/2] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[19:05:48]	[Step 2/2] 	at java.lang.Thread.run(Thread.java:745)
[19:05:48]	[Step 2/2] [error] (evaluation-ui/*:dockerPush) There was a problem pushing tag "e175f15e7bd2df7c7b084b5d840a2874d4436f89", stopping.
```

Note that the tag being pushed was "cleaned up" immediately prior to the start of the push, causing an error.

**Comment by jkinkead** [comment]
This is a bug in `dockerBuild`; fix forthcoming.

**Comment by jkinkead** [comment]
This will be fixed in `1.5.1`.
